store/postgres/src/store.rs

```diff
@@ -236,6 +236,10 @@ impl BasicStore for Store {
                 .offset(range.skip as i64);
         }
 
+        {
+            debug!(self.logger, "find"; "Postgres query" => format!("{:?}", debug_query::<Pg, _>(&diesel_query)));
```

Comment:
This line is too long. `rustfmt` can't properly format the semicolon syntax. Do it manually:
```rust
debug!(self.logger, "find";
       "sql" => format("{:?}", debug_query::<Pg, _>(&diesel_query)));
```
Also note that we're using single-word labels for keys in the logged key/value pairs ("sql" instead of "Postgres query").